benches/my_benchmark.rs

```diff
@@ -113,7 +113,7 @@ pub fn bench_collect_uint(c: &mut Criterion) {
                 let mut bm = croaring::Bitmap::create();
                 bm.add_range(0..batch_size as u64);
                 b.iter(|| {
-                    let _ = &bm.to_vec().iter().map(|v| *v as usize).collect::<Vec<_>>();
```

Comment:
This is probably the biggest question in the PR — I don't know why the old way was allocating a `Vec`, then iterating through it, then collecting a second `Vec`.

Reply:
Looks like an oversight, thanks 🤷 

I am pretty sure I wrote most of these benches in various meetings 😂